ui/notification/utils.go

```diff
@@ -0,0 +1,22 @@
+package notification
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+)
+
+func getAbsolutePath() (string, error) {
```

Comment:
I don't think any utils.go file is needed in this package since it's just this function and it's used only in system.go